### PR TITLE
lux: update to 0.24.1

### PR DESCRIPTION
**Pull request by herbygillot**
###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with ~`sudo port -vst install`~ `sudo port -d destroot`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: net/lux/Portfile [hunk 1]
```diff
@@ -3,225 +3,24 @@
...
 description         fast and simple video downloader
 
 long_description    Lux is a ${description} built with Go.
 
-checksums           ${distname}${extract.suffix} \
-                        rmd160  ef8fe3e8e8b4df2170f318bbd39ad504bece6679 \
-                        sha256  b3e70a63ad5053b37f95f717c97df6d8d57e4b63abf46df88a06d1cca4fa5443 \
-                        size    103624
-
-go.vendors          gopkg.in/yaml.v3 \
```
Comment (l2dy) [inline net/lux/Portfile:20]: Why was this removed? Is vendoring dependencies no longer possible?

Comment (herbygillot) [reply net/lux/Portfile:20]: Ran into some issues trying to vendor this version, and just chose not to deal with the struggle of getting it to work.
